### ISSUE TITLE
[Feature Request] Infinity Containers keep contents on break

### ISSUE THREAD

**Issue by ouroborus** [CJ-MC-Mods/IBHSTD#32]
#### Overloaded Version: 1.12.1-0.0.44

This is either a bug report or a feature request. Currently the various infinite storage blocks lose their contents when broken. It'd be great if they instead kept their contents instead.

**Comment by cjm721** [comment]
That is intended. Was planning on adding a special block to move them at some point. If you use the hyper transfer nodes you can move their entire contents in one tick so if you really need to move it just swap it to another one for a minute.

**Comment by ouroborus** [comment]
My current solution is a cardboard box (Mekanism). Certainly cheaper than the transfer nodes.

The main thing is that the containers already cost a fortune. Having them lose their inventory on accidental breaking (their hardness is pretty low) is just putting salt in the wound.

**Comment by cjm721** [comment]
Want to disincentivize moving them outside of nodes as the nodes are the least server intensive way to move their contents.

They are expensive to make because of their storage. If they are going to keep their contents need to be more expensive / have a better disincentivize for them then as one time cost is really not that much of a obstruction.

**Comment by ouroborus** [comment]
Fair enough. Then maybe just increase their mining hardness? Maybe 5-10 times whatever it is now.

**Comment by cjm721** [comment]
If you don't want to break them on accident increasing the hardness is not really a good method (multi-tool instantly breaks everything for example, or draconic evolution pick which has an extremely fast break speed also).

**Comment by ouroborus** [comment]
I would argue that both the multi-tool and the DE pick are considered dangerous anyway and can't be considered representative. They plow through pretty much everything to fast to be considered controlled. I'm thinking more along the lines of tools between obsidian picks and the atomic disassembler (Mekanism) on fast mode. On the other hand, you make a good point in that a player who has the resources to make overloaded items probably already has DE stuff if they're available.

**Comment by cjm721** [comment]
Idea though to add to my list of things is a block/item combination that prevents all breaking of blocks in an area while active (with opt in to make it not pvp usable)
